src/test/java/com/amplitude/HttpCallTest.java

```diff
@@ -238,4 +270,16 @@ private void verifyConnectionOption(HttpsURLConnection connection) throws Protoc
     verify(connection, times(1)).setReadTimeout(Constants.NETWORK_TIMEOUT_MILLIS);
     verify(connection, times(1)).setDoOutput(true);
   }
+
+  private JSONObject getMockResponse(
```

Comment:
nit: this could be `getMockSuccessResponse`

Reply:
I was thinking that too but decided to make the status code configurable for possible reuse in other tests.